mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionsProvider.scala

```diff
@@ -237,7 +237,8 @@ class CompletionsProvider(
                   case IndexedContext.Result.InScope =>
                     mkItem(ident, ident.backticked)
                   case _ => mkWorkspaceItem(ident, sym.fullNameBackticked)
-      case CompletionValue.NamedArg(label, _) => mkItem(ident, ident)
+      case CompletionValue.NamedArg(label, _) =>
+        mkItem(ident, ident.replace("$", "\\$")) // escape $ for snippet
```

Comment:
I'm wondering if it might be worth escaping `$` for all completion items.
`$` char might come not only from NamedArg

Reply:
That's fair enough, but we have to carefully escape `$` because we can't escape in [snippet syntax of LSP](https://github.com/microsoft/language-server-protocol/blob/main/snippetSyntax.md).
I couldn't figure out what is the best way to escape `$` from whole completion items without breaking the snippet syntax. Therefore, let me limit to NamedArg here :bow: